pkg/k8shandler/configmaps.go

```diff
@@ -13,28 +13,33 @@ import (
 	//"github.com/sirupsen/logrus"
 )
 
-// CreateOrUpdateConfigMaps ensures the existens of ConfigMaps with Elasticsearch configuration
+// CreateOrUpdateConfigMaps ensures the existence of ConfigMaps with Elasticsearch configuration
 func CreateOrUpdateConfigMaps(dpl *v1alpha1.Elasticsearch) (string, error) {
 	owner := asOwner(dpl)
-	var configMapName string
+	configMapName := dpl.Spec.ConfigMapName
 	if dpl.Spec.ConfigMapName == "" {
 		configMapName = dpl.Name
-	} else {
-		configMapName = dpl.Spec.ConfigMapName
 	}
 
 	// TODO: take all vars from CRD
-	pathData := "- /elasticsearch/persistent/"
-	err := createOrUpdateConfigMap(configMapName, dpl.Namespace, dpl.Name, defaultKibanaIndexMode, pathData, false, dpl.Spec.Secure.Disabled, owner, dpl.Labels)
+	kibanaIndexMode, err := KibanaIndexMode("")
+	if err != nil {
+		return "", err
+	}
+	esUnicastHost := EsUnicastHost("es")
+	allowClusterReader := AllowClusterReader()
+	rootLogger := RootLogger()
```

Comment:
@ewolinetz  Should these vars be configurable? How?

Reply:
I think we should really just be using the file logger as the root logger...

@richm do we need to use different values for CI testing?

Reply:
@ewolinetz  `esUnicastHost := EsUnicastHost("es")`
Are we still using this pattern?
```
func EsUnicastHost(clusterName string) string {
	esComponent := "es"
	if clusterName == "elasticsearch-infra" {
		esComponent = "es-ops"
	}
	return fmt.Sprintf("logging-%s-cluster", esComponent)
}
```

Reply:
the other components are created such that they expect it will be one of `elasticsearch`, `elasticsearch-app`, or `elasticsearch-infra`

https://github.com/openshift/cluster-logging-operator/blob/master/pkg/k8shandler/visualization.go#L53
https://github.com/openshift/cluster-logging-operator/blob/master/pkg/k8shandler/visualization.go#L63
https://github.com/openshift/cluster-logging-operator/blob/master/pkg/k8shandler/visualization.go#L73